### PR TITLE
Localization Support

### PR DESCRIPTION
**Pull request by TheJF**
This adds localization support to the stacked chart for all locales defined by the d3 project.

The chart relies on the `lang` attribute of the HTML tag to figure out what locale it should use. If it has none, it will default to `en-US`.

Currently, the locales are JS files as they have been lifted from the d3 project. If necessary, they can be converted to JavaScript (or even JSON)

### CODE REVIEW

File: angularD3/locale/index.coffee [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+localeDefinitions["it-it"] = require("./it-IT")
+localeDefinitions["ja-jp"] = require("./ja-JP")
+localeDefinitions["ko-kr"] = require("./ko-KR")
+localeDefinitions["mk-mk"] = require("./mk-MK")
+localeDefinitions["nl-nl"] = require("./nl-NL")
+localeDefinitions["pl-pl"] = require("./pl-PL")
+localeDefinitions["pt-br"] = require("./pt-BR")
+localeDefinitions["ru-ru"] = require("./ru-RU")
+localeDefinitions["sv-se"] = require("./sv-SE")
+localeDefinitions["zh-cn"] = require("./zh-CN")
```
Comment (chrisnicola) [inline angularD3/locale/index.coffee]: We should make it possible to not require every single locale definition file. Just the ones you need to use.

Comment (TheJF) [inline angularD3/locale/index.coffee]: Agreed, but I couldn't wrap my head around how to set that up.

File: angularD3/services/d3locale.coffee [hunk 2]
```diff
@@ -0,0 +1,13 @@
+d3 = require('d3')
+localeDefinitions = require('../locale')
+
+angular.module('ad3').service 'd3locale', [ ->
+  @getD3Locale = (locale = false) ->
```
Comment (chrisnicola) [inline angularD3/services/d3locale.coffee]: Should have a configurable provider to allow the translations to be configured.

**Comment by chrisnicola** [comment]
This all looks good now.

**Pull request merged**